packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -204,7 +204,7 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
   const onRenderAttachmentDownloads = useCallback(
     (userId: string, message: ChatMessage) => (
       <_AttachmentDownloadCards
-        attachments={message.files}
+        attachments={message.attachments || []}
```

Comment:
Should this be an empty array?

Reply:
logic is updated to return empty div if attachment is empty